app/forms/placement_form.rb

```diff
@@ -28,7 +28,11 @@ def initialize(placements_form:, placement:, destroy: false)
     @placement = placement
     @destroy = destroy
     self.attributes = placement.attributes.symbolize_keys.slice(*FIELDS)
-    self.school_search = placement.school_search || placement.school&.name
+    self.school_search = placement.school_search
+  end
+
+  def initialise_school_search_for_edit
```

Comment:
not something i often say, but perhaps this warrants a code comment, as it requires an awful lot of context to understand why it's doing what it's doing.